### ISSUE TITLE
After i trained 100000batches, my valid dataset map is 0.75, but when i test some images, just a little objects can be detected

### ISSUE THREAD

**Issue by Chinajuedui** [AlexeyAB/darknet#6306]
Model is Yolov4-tiny.The number of classes is 3.Avg loss was 1.96* when reached 1000 batches and 0.67* when reached 20000 batches, their map was 0.26 and 0.69.The number of train images is 150000 and the number of valid images is 10000.The number of marked bounding box of class one is 190000.The number of marked bounding box of class two is 42000. The number of marked bounding box of class three is 70000. Should i balance marked bounding box?My cfg file is as fellows:

batch=240
subdivisions=1
width=416
height=416
channels=3
momentum=0.9
decay=0.0005
angle=0
saturation = 1.5
exposure = 1.5
hue=.1

learning_rate=0.000261
burn_in=4000
max_batches = 200000
policy=steps
steps=100000,150000
scales=.1,.1
Thank you for your help.
